feedthefox/base/templates/base.jinja

```diff
@@ -14,39 +14,46 @@
-                    data-toggle="collapse" data-target="#navbar-collapse" aria-expanded="false">
-              <span class="sr-only">Toggle navigation</span>
-              <span class="icon-bar"></span>
-              <span class="icon-bar"></span>
-              <span class="icon-bar"></span>
-            </button>
-            <div class="navbar-brand">
-              <a href="{{ url('home') }}">
-                <i class="fxfamilynav-logo hidden-sm hidden-xs">Firefox</i>
-                <span class="logo-wordmark">Contribution Hub</span>
-              </a>
+      <header>
+
+        <nav class="navbar navbar-default {% block navbar_sticky %}navbar-sticky{% endblock %}">
+          <div class="container">
+            <div class="navbar-header">
+              <button type="button" class="navbar-toggle collapsed"
+                      data-toggle="collapse" data-target="#navbar-collapse" aria-expanded="false">
+                <span class="sr-only">Toggle navigation</span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+              </button>
+              <div class="navbar-brand">
+                <a href="{{ url('home') }}">
+                  <i class="fxfamilynav-logo hidden-sm">Firefox</i>
+                  <span class="logo-wordmark">{% block navbar_wordmark %}Firefox OS {% endblock %}Contribution Hub</span>
+                </a>
+              </div>
             </div>
-          </div>
 
-          <div class="collapse navbar-collapse navbar-left" id="navbar-collapse">
-            <ul class="nav navbar-nav">
-              <li><a href="{{ url('devices') }}">Devices</a></li>
-              <li><a href="{{ url('dashboard') }}">Dashboard</a></li>
-              <li><a href="#">
-                <span class="glyphicon glyphicon-user" aria-hidden="true"></span> Connect
-              </a></li>
-            </ul>
-          </div>
+            <div class="collapse navbar-collapse navbar-left" id="navbar-collapse">
+              <ul class="nav navbar-nav">
+                <li><a href="{{ url('devices') }}">Devices</a></li>
+                <li><a href="{{ url('dashboard') }}">Dashboard</a></li>
+                <li><a href="{{ url('profile') }}">
+                  <span class="glyphicon glyphicon-user" aria-hidden="true"></span>
+                  <span class="profile-word">Connect</span>
+                </a></li>
+              </ul>
+            </div>
 
-          <div id="tabzilla" class="hidden-xs">
-            <a href="https://www.mozilla.org/">Mozilla</a>
+            <div id="tabzilla" class="hidden-xs">
+              <a href="https://www.mozilla.org/">Mozilla</a>
+            </div>
           </div>
-        </div>
-      </nav>
+        </nav>
+
+        {% block masthead %}
+        {% endblock %}
+      </header>
 
       <div class="page">
         {% block main %}
```

Comment:
Because allauth templates are using `content` as placeholder, lets change this here (far fewer changes here than editing all the templates in allauth).